src/main/java/frc/robot/constants/Constants2023.java

```diff
@@ -32,6 +32,9 @@ public class Constants2023 extends GameConstants {
     public static final double SHOOTER_MOTOR_LOW_SPEED = 0.6;
     public static final double SHOOTER_MOTOR_HIGH_SPEED = 1;
 
+    public static final double SHOOTER_MOTOR_RIGHT_AMP_SPEED = 0.25;
+    public static final double SHOOTER_MOTOR_LEFT_AMP_SPEED = 0.25;
```

Comment:
Also I don't think any of this should have been in this PR.

Reply:
Deleted